myuw/static/vue/components/_common/course/course.vue

```diff
@@ -131,22 +152,6 @@ export default {
     };
   },
   computed: {
-    instructors() {
-      const seenUWRegId = new Set();
-      return this.section.meetings.map(
-          (s) => s.instructors || [],
-      ).flat().filter((i) => {
-        if (seenUWRegId.has(i.uwregid)) return false;
-        seenUWRegId.add(i.uwregid);
-        return true;
-      }).sort((i1, i2) => {
-        if (i1.surname < i2.surname) return -1;
-        if (i1.surname > i2.surname) return 1;
-        return 0;
-      });
-    },
-  },
-  methods: {
     sectionFormattedDates(section) {
```

Comment:
this should be inside methods not computed as it takes a parameter and computed can't